src/apic-extension/azext_apic_extension/command_patches.py

```diff
@@ -345,7 +345,6 @@ class ShowIntegrationExtension(DefaultWorkspaceParameter, ShowIntegration):
 
 @register_command(
     "apic integration create apim",
-    is_preview=True,
```

Comment:
the integration feature should still in preview,  so we need to keep this flag. The import apim functionality should be in GA status per my understanding. Let's double confirm with PM.